### DISCUSSION TITLE
major issue with performance of our client-mysql

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#681] [Q&A]
![image](https://user-images.githubusercontent.com/30953478/229083380-f312ea1d-2c3e-4309-9b01-2814a56b9672.png)
logs of one product being loaded, for note we use mariadb, there were 40-50 tcp connections in 10 seconds to make it
not sure specifics but instance of pim running in the container, also we use separate user than www-data in it for application to work 

The issue we've got is atrocore client connection to mysql not being persistent. + I suppose each time i open a page(single product):
  - each part of the data product acquire through api - gets its session verified, that triggers mysql query to do so, then bit of data retrieved one at a time

devops people say it is because of mysql connection not being persistent with php, in any regard thanks wheather you could help on this matter

**Comment by rratsun** [comment]
Hello,

We does not [support](https://github.com/atrocore/atrocore#requirements) MariaDB. Use MySQL instead.

**Reply by zssvaidar** [reply]
my bad, it's correct we use mysql,
Let me clarify what we've encountered is when used tcp connection over stocket to connect mysql database, load time of each product become at least 8-12 sec and 15-20sec with full productAttributes

**Reply by rratsun** [reply]
I've created an internal issue for this. About results, we will inform you later.
